<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/ico/pizza.ico</normaloff>images/ico/pizza.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #2D9CDB;</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>141</width>
     <height>261</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>124</width>
     <height>247</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>436</width>
     <height>2342352</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #2D9CDB;
border: 1px solid #DDDDDD;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="profileLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{font-family: Arial;
font-style: normal;
font-weight: normal;
font-size: 12px;
line-height: 14px;
color: #000000;
background: #FFFFFF;}
QLabel:hover{
background:#e6f3ff;
}</string>
    </property>
    <property name="text">
     <string>Профиль</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>441</width>
     <height>321</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>441</width>
     <height>301</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>441</width>
     <height>321</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
color: #FFFFFF;
background: #006EAC
;}

QPushButton:hover{
background-color:#005da7;
}

QPushButton:pressed{
background-color:#2D9CDB;
}</string>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <widget class="QLabel" name="mainLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>21</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>21</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Arial;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 21px;
color: #000000;</string>
    </property>
    <property name="text">
     <string>Настройка профиля</string>
    </property>
   </widget>
   <widget class="QLabel" name="fioLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>14</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>14</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Arial;
font-style: normal;
font-weight: normal;
font-size: 12px;
line-height: 14px;
color: #000000;</string>
    </property>
    <property name="text">
     <string>Ф.И.О</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fioEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="loginLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>14</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>14</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Arial;
font-style: normal;
font-weight: normal;
font-size: 12px;
line-height: 14px;
color: #000000;</string>
    </property>
    <property name="text">
     <string>Логин</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="loginEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="passwordLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>14</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>14</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Arial;
font-style: normal;
font-weight: normal;
font-size: 12px;
line-height: 14px;
color: #000000;</string>
    </property>
    <property name="text">
     <string>Пароль</string>
    </property>
   </widget>
   <widget class="QLabel" name="positionLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>71</width>
      <height>14</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>14</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>14</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Arial;
font-style: normal;
font-weight: normal;
font-size: 12px;
line-height: 14px;
color: #000000;</string>
    </property>
    <property name="text">
     <string>Должность</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="positionEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>141</width>
     <height>41</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>141</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:none;
color: #FFFFFF;
background: #006EAC
;}

QPushButton:hover{
background-color:#005da7;
}

QPushButton:pressed{
background-color:#2D9CDB;
}</string>
   </property>
   <property name="text">
    <string>Вернуться назад</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
